[CI] MixedClusterClientYamlTestSuiteIT test {p0=tsdb/150_runtime_fields/can't shadow metrics} failing

**Build Scans:**
- [elasticsearch-periodic #11962 / 8.0.1_bwc](https://gradle-enterprise.elastic.co/s/6zzatomlc35ai)
- [elasticsearch-periodic #11962 / 8.0.1_bwc](https://gradle-enterprise.elastic.co/s/3bky4gntl4ol6)
- [elasticsearch-periodic #11962 / 7.17.29_bwc](https://gradle-enterprise.elastic.co/s/cinusre3tsugs)
- [elasticsearch-periodic #11958 / 8.0.1_bwc](https://gradle-enterprise.elastic.co/s/t5qxmidsphsfy)
- [elasticsearch-periodic #11958 / 7.17.29_bwc](https://gradle-enterprise.elastic.co/s/hn6zzuh4r253q)
- [elasticsearch-periodic #11954 / 8.0.1_bwc](https://gradle-enterprise.elastic.co/s/m532lbmqxxclw)
- [elasticsearch-periodic #11954 / 7.17.29_bwc](https://gradle-enterprise.elastic.co/s/btdiytthixime)
- [elasticsearch-periodic #11950 / 8.0.1_bwc](https://gradle-enterprise.elastic.co/s/itz6b6alcibds)
- [elasticsearch-periodic #11950 / 7.17.29_bwc](https://gradle-enterprise.elastic.co/s/wm7rvwqlpq7nq)
- [elasticsearch-periodic #11946 / 8.0.1_bwc](https://gradle-enterprise.elastic.co/s/eevuel2uqlmb6)

**Reproduction Line:**
```
./gradlew ":qa:mixed-cluster:v8.0.1#mixedClusterTest" -Dtests.class="org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT" -Dtests.method="test {p0=tsdb/150_runtime_fields/can't shadow metrics}" -Dtests.seed=F4679A46C62DDE8A -Dtests.bwc=true -Dtests.locale=agq-CM -Dtests.timezone=CST -Druntime.java=25
```

**Applicable branches:**
8.19

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('test%20%7Bp0%3Dtsdb%2F150_runtime_fields%2Fcan't%20shadow%20metrics%7D'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: Failure at [tsdb/150_runtime_fields:134]: expected [2xx] status code but api [indices.create] returned [400 Bad Request] [{"error":{"root_cause":[{"type":"illegal_argument_exception","reason":"unknown setting [index.time_series.end_time] please check that any required plugins are installed, or check the breaking changes documentation for removed settings","stack_trace":"org.elasticsearch.ElasticsearchException$1: unknown setting [index.time_series.end_time] please check that any required plugins are installed, or check the breaking changes documentation for removed settings\n\tat org.elasticsearch.server@8.19.13-SNAPSHOT/org.elasticsearch.ElasticsearchException.guessRootCauses(ElasticsearchException.java:709)\n\tat org.elasticsearch.server@8.19.13-SNAPSHOT/org.elasticsearch.ElasticsearchException.generateFailureXContent(ElasticsearchException.java:637)\n\tat org.elasticsearch.server@8.19.13-SNAPSHOT/org.elasticsearch.rest.RestResponse.build(RestResponse.java:2
[truncated]
```

**Issue Reasons:**
- [8.19] 8 consecutive failures in step 8.0.1_bwc
- [8.19] 7 consecutive failures in step 7.17.29_bwc
- [8.19] 15 failures in test test {p0=tsdb/150_runtime_fields/can't shadow metrics} (10.7% fail rate in 140 executions)
- [8.19] 8 failures in step 8.0.1_bwc (100.0% fail rate in 8 executions)
- [8.19] 7 failures in step 7.17.29_bwc (100.0% fail rate in 7 executions)
- [8.19] 7 failures in pipeline elasticsearch-periodic (100.0% fail rate in 7 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/tsdb/150_runtime_fields.yml
+++ b/rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/tsdb/150_runtime_fields.yml
@@ -61,6 +61,9 @@ tsdb_execute_painless_api:
 
 ---
 can't shadow dimensions:
+  - requires:
+      cluster_features: [ "gte_v8.1.0" ]
+      reason: introduced in 8.1.0
   - do:
       indices.create:
         index: test
@@ -131,6 +134,9 @@ can't shadow dimensions:
 
 ---
 can't shadow metrics:
+  - requires:
+      cluster_features: [ "gte_v8.1.0" ]
+      reason: introduced in 8.1.0
   - do:
       indices.create:
         index: test

